### DISCUSSION TITLE
Multiple local domains?

### DISCUSSION THREAD

**Discussion by aolko** [php/frankenphp#684] [Q&A]
Can frankenphp be configured to serve multiple "apps" on multiple domains, available locally in a standalone app?
```
# the "apps" can be either located in subdirs, or have domain dirs
/
L /dashboard/ -> myapp.local
L /wiki/ -> wiki.myapp.local
L /docs/ -> docs.myapp.local
L /forum/ -> myforum.local

/
L /myapp.local/
L /wiki.myapp.local/
L /docs.myapp.local/
L /myforum.local/
```

**Comment by withinboredom** [comment]
Yes, but if you need to rely on filesystem ACL to provide isolation between them, this isn't possible because Frankenphp uses threads, not processes.

**Reply by Chris53897** [reply]
@withinboredom Could you please point out some more details, how to configure this?

**Comment by dah-HUB** [comment]
Hi @aolko 

Plz, Did you solve this issue ?

I have the same, plus, managing PHP versions -> https://github.com/dunglas/frankenphp/discussions/721

I don't know if this is a red flag to not use this app or it's just a missing documentation ?

Thx & Good Luck

**Reply by aolko** [reply]
Not yet since there's no static php with [all the modules I need for windows](https://github.com/crazywhalecc/static-php-cli/discussions/389). [If only franken dev would wake up asap](https://github.com/dunglas/frankenphp/discussions/420) 🤔.

**Reply by withinboredom** [reply]
@dah-HUB Have you tried searching previous issues?

https://github.com/dunglas/frankenphp/issues/219#issuecomment-1739968127 seems to show how it could be done.

Any contributions to the docs is welcome.
